knip.config.ts

```diff
@@ -0,0 +1,14 @@
+import { type KnipConfig } from 'knip';
```

Comment:
We usually use a different import syntax.

```suggestion
import type { KnipConfig } from 'knip';
```

Reply:
Solved ✅